app/src/main/res/values/styles.xml

```diff
@@ -363,6 +379,16 @@
         <item name="android:clickable">true</item>
     </style>
 
+    <style name="ListItem" parent="@android:style/TextAppearance">
```

Comment:
This style seems almost identical to `OverflowMenuItem`. Can one be a child of the other?

Reply:
Done!